plugins/outputs/nats/nats.go

```diff
@@ -240,7 +243,17 @@ func (n *NATS) Init() error {
 		if !choice.Contains(n.Subject, n.Jetstream.Subjects) {
 			n.Jetstream.Subjects = append(n.Jetstream.Subjects, n.Subject)
 		}
+
 		var err error
+		if n.Jetstream.AsyncAckTimeout == "" {
+			n.Jetstream.AsyncAckTimeoutDuration = 5 * time.Second
+		} else {
+			n.Jetstream.AsyncAckTimeoutDuration, err = time.ParseDuration(n.Jetstream.AsyncAckTimeout)
+			if err != nil {
+				return fmt.Errorf("failed to parse async_ack_timeout config: %w", err)
+			}
+		}
```

Comment:
No. This is done automatically when using `config.Duration` as type for `AsyncAckTimeout`. You can then simply cast this to `time.Duration`... The default should be set in `init` when registering the plugin.

Reply:
@srebhan, thanks for good feedback, have now updated PR with your suggestions.